### PR TITLE
Change occurrences of frontrow to freckle

### PR DESCRIPTION
**Pull request by eborden**

### CODE REVIEW

File: js/main.js [hunk 1]
```diff
@@ -37,20 +35,20 @@
...
-        return "It appears that your network is blocking a domain required by Front Row. Learn more about it at the " + blockedPageLink;
+        return "It appears that your network is blocking a domain required by Freckle. Learn more about it at the " + blockedPageLink;
     }
   }
 
   function launchAllChecks () {
     var baseAPIsUrl = getBaseAPIsUrl();
     var apiUrl      = baseAPIsUrl + "2";
-    var assetsUrl   = "https://classroom-assets.frontrowed.com";
+    var assetsUrl   = "https://classroom-assets.freckle.com";
```
Comment (jdreaver) [inline js/main.js]: This still needs to be https://classroom-assets.frontrowed.com

**Comment by eborden** [comment]
Okay, this should be good. I was able to run it on my computer with only CORS failures from mixpanel.

File: js/main.js [hunk 2]
```diff
@@ -357,17 +358,18 @@
...
       case "trackjs":     return "Sending information to TrackJs";
       case "mixpanel":    return "Sending information to MixPanel";
       case "imgix":       return "Fetching information from Imgix";
       case "tts":         return "Fetching jQuery library";
-      case "browser":     return "Use Front Row on my browser";
-      case "screensize":  return "Use Front Row on my device";
+      case "browser":     return "Use Freckle on my browser";
+      case "screensize":  return "Use Freckle on my device";
       case "timeout":     return "Connection timeout";
+      case "old-imgix":       return "Fetching information from Imgix";
```
Comment (joris974) [inline js/main.js]: [non-blocking] alignment is off.

**Review by joris974 — APPROVED** [review]

File: js/main.js (hunk 2 above)
Comment (eborden) [reply js/main.js]: alignment :sob:

File: js/main.js [hunk 3]
```diff
@@ -52,6 +51,7 @@
     var apiUrl      = baseAPIsUrl + "2";
     var assetsUrl   = "https://classroom-assets.frontrowed.com";
     var imgixUrl    = "https://frontrow-image-assets.imgix.net";
+    var oldImgixUrl    = "https://frontrow-image-assets.imgix.net";
```
Comment (jdreaver) [inline js/main.js]: Is this just future proofing? This is the same as the previous URL.

**Review by jdreaver — APPROVED** [review]

File: js/main.js (hunk 3 above)
Comment (eborden) [reply js/main.js]: Lollllll, just me being dumb.

**Pull request merged**